test/backup.ts

```diff
@@ -164,6 +167,7 @@ describe('Backup', () => {
 
         assert.notStrictEqual(config.reqOpts, QUERY);
         assert.deepStrictEqual(QUERY, ORIGINAL_QUERY);
+        assert.deepStrictEqual(config.headers, resourceHeader);
```

Comment:
To test that we're sending the right header, we should compare to "instance.resourceHeader_".